junit/src/main/java/org/kaazing/k3po/junit/rules/Latch.java

```diff
@@ -139,9 +139,10 @@ boolean hasException() {
 
     void notifyException(Exception exception) {
         this.exception = exception;
-        prepared.countDown();
-        startable.countDown();
-        finished.countDown();
+        // Commented because of issue https://github.com/k3po/k3po/issues/391
```

Comment:
What we need to check is what was the role of the commented lines. I don't see any.